install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -73,8 +74,10 @@ spec:
               endpoint:
                 properties:
                   external:
+                    description: The endpoint external to the cluster within which the Broker is running, which the Meshery Server will connect to
                     type: string
                   internal:
+                    description: The endpoint internal to the cluster within which the Broker is running, will be a locally addressable IP address or DNS hostname (e.g.`broker.busybox.svc.cluster-domain.example`, `10.96.168.204:4222`)
```

Comment:
Please clarify whether the port is required here or is separately defined.

Reply:
Regarding the port, there was an oversight on my part not reading the Meshery Docs extensively on this description, I have rectified this with a new description which extends the external endpoint not the internal.